ARM Templates/FailoverReprotect/azuredeploy.json

```diff
@@ -0,0 +1,102 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "identity": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM ID of a user managed identity that has contributor access to the resource group being deployed."
+            }
+        },
+        "vaultSubscriptionId": {
+            "type": "String",
+            "metadata": {
+                "description": "Subscription Id of the Recovery Services Vault."
+            }
+        },
+        "vaultName": {
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Recovery Services Vault to be used."
+            }
+        },
+        "vaultResourceGroupName": {
+            "type": "String",
+            "metadata": {
+                "description": "Resource Group Name of the Recovery Services Vault."
+            }
+        },
+        "primaryRegion": {
+            "type": "String",
+            "metadata": {
+                "description": "The region where the original source virtual machines are deployed."
+            }
+        },
+        "recoveryRegion": {
+            "type": "String",
+            "metadata": {
+                "description": "The designated disaster recovery region where virtual machines would be brought up after failover."
+            }
+        },
+        "sourceVmARMIdsCSV": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM IDs of the Source VMs separated by comma."
+            }
+        },
+        "recoveryStagingStorageAccount": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM ID of the storage account to be used to cache replication data in the DR region (for Reprotect operation)."
+            }
+        },
+        "recoveryReplicaDiskAccountType": {
+            "defaultValue": "Standard_LRS",
+            "allowedValues":[
```

Comment:
as per service, the allowed values are:         
        Standard_LRS,

        Premium_LRS,

        StandardSSD_LRS

Reply:
Okay. Will update. Thanks!